content/webapp/services/prismic/fetch/articles.ts

```diff
@@ -6,10 +6,319 @@ import { ContentType } from '../link-resolver';
+        non-repeat {
+          title
+        }
+        repeat {
+          link
+          linkText
+        }
+      }
+      ...on imageList {
+        non-repeat {
+          listStyle
+          description
+        }
+        repeat {
+          title
+          subtitle
+          image
+          caption
+          description
+        }
+      }
+    }
+    contributors {
+      ...contributorsFields
+      role {
+        ...roleFields
+      }
+      contributor {
+        ... on people {
+          ...peopleFields
+        }
+        ... on organisations {
+          ...organisationsFields
+        }
+      }
+    }
+    series {
+      series {
+        ...seriesFields
+        schedule {
+          ...scheduleFields
+        }
+        contributors {
+          ...contributorsFields
+          role {
+            ...roleFields
+          }
+          contributor {
+            ... on people {
+              ...peopleFields
+            }
+            ... on organisations {
+              ...organisationsFields
+            }
+          }
+        }
+        promo {
+          ... on editorialImage {
+            non-repeat {
+              caption
+              image
+            }
+          }
+        }
+      }
+    }
+    outroResearchItem {
+      ... on events {
+        ...eventsFields
+      }
+      ... on exhibitions {
+        title
+      }
+      ... on books {
+        title
+      }
+      ... on articles {
+        title
+      }
+      ... on webcomics {
+        title
+      }
+      ... on series {
+        title
+      }
+      ... on event-series {
+        title
+      }
+      ... on pages {
+        title
+      }
+    }
+    outroReadItem {
+      ... on events {
+        ...eventsFields
+      }
+      ... on exhibitions {
+        title
+      }
+      ... on books {
+        title
+      }
+      ... on articles {
+        title
+      }
+      ... on webcomics {
+        title
+      }
+      ... on series {
+        title
+      }
+      ... on event-series {
+        title
+      }
+      ... on pages {
+        title
+      }
+    }
+    outroVisitItem {
+      ... on events {
+        ...eventsFields
+      }
+      ... on exhibitions {
+        title
+      }
+      ... on books {
+        title
+      }
+      ... on articles {
+        title
+      }
+      ... on webcomics {
+        title
+      }
+      ... on series {
+        title
+      }
+      ... on event-series {
+        title
+      }
+      ... on pages {
+        title
+      }
+    }
+    promo {
+      ... on editorialImage {
+        non-repeat {
+          caption
+          image
+        }
+      }
+    }
+    seasons {
+      season {
+        ... on seasons {
+          title
+          promo {
+            ... on editorialImage {
+              non-repeat {
+                caption
+                image
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}`.replace(/\n(\s+)/g, '\n');
```

Comment:
This is copied directly from `articles.js`. I'm not entirely sure what it does, but it was part of the original implementation of `getArticles()` and I didn't port it properly when I migrated that to the new fetchers. I'm not entirely sure what it does tbh, but since it was here originally I'm going to keep it for now.